modules/troubleshooting-disabling-autoreboot-mco.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+[id="troubleshooting-disabling-autoreboot-mco_{context}"]
+= Disabling Machine Config Operator from autorebooting
+
+When you use the Machine Config Operator to modify an NTP, registry, secret, SSH key, or other resource, {op-system-first} must be rebooted for the change to take effect. To avoid unwanted disruptions, you can modify the machine config pool to prevent autorebooting after the Operator makes changes to the machine config.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+* You have root access in {product-title}.
+
+.Procedure
+To pause the autoreboot process after machine config changes are applied:
+
+. As root, modify the `spec.paused` field to `true`.
++
+.Control plane (master) nodes
+[source,terminal]
+----
+# oc patch --type=merge --patch='{"spec":{"paused":true}}' machineconfigpool/master
+----
++
+.Worker nodes
+[source,terminal]
+----
+# oc patch --type=merge --patch='{"spec":{"paused":true}}' machineconfigpool/worker
+----
+
+. Verify whether the machine config pool is paused or unpaused.
++
+.Control plane (master) nodes
+[source,terminal]
+----
+oc get machineconfigpool/master --template='{{.spec.paused}}'
+----
++
+.Worker nodes
+[source,terminal]
+----
+# oc get machineconfigpool/worker --template='{{.spec.paused}}'
+----
++
+If the `spec.paused` field is `true` in the output, the machine config pool is paused. If the field is `false`, it is unpaused.
+
+To unpause the autoreboot process:
```

Comment:
I think these need to be two separate procedures, or somehow handled differently. I don't think we normally allow a second set of steps in a procedure like this.

Reply:
I went back and forth on this. Ultimately, I left as a single procedure and made the `unpause` task an alternative step. I didn't love the idea of a separate procedure that is mostly to reverse the previous procedure. IIUC, a user would only unpause the process if they had previously paused it by following the previous step, so I didn't want the content decoupled.